playbooks/deploy-rock.yml

```diff
@@ -476,13 +476,14 @@
       src: filebeat.yml.j2
       dest: /etc/filebeat/filebeat.yml
     notify: Restart Filebeat
+    when: with_fsf or with_suricata
 
   - name: Enable and start Filebeat
     service:
       name: filebeat
       state: "{{ 'started' if enable_filebeat else 'stopped' }}"
```

Comment:
This should be dependent on `enable_fsf` or `enable_suricata` maybe. You deleted this variable in the config.